Homework/Marianna/ClassesObjectsInheritance/ShapeClass/Circle.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShapeClass
+{
+    class Circle : TwoDimensionalShape
+    {
+        public Circle(double a) : base(a)
+        {
+
+        }
+        public override double Area
+        {
+            get { return Math.PI * Math.Pow(base.Side, 2); }
+        }
+        public override string ToString()
+        {
+            return string.Format("It is a circle! " + base.ToString());
+        }
```

Comment:
It's better to have at least one empty line between methods, properties and class/struct/enum definitions.
In essence: if you see a closing curly bracket - add an empty line after it.

Reply:
Exceptions to this: if-else chains, closing bracket after closing bracket (e.g. end of an if, then end of a for loop, then end of a method)